packages/amesos2/test/solvers/MUMPS_test.xml

```diff
@@ -31,6 +31,7 @@
 	  <Parameter name="LocalOrdinal" type="string" value="int"/>
 	  <Parameter name="GlobalOrdinal" type="strig" value="int"/>
```

Comment:
```suggestion
	  <Parameter name="GlobalOrdinal" type="string" value="int"/>
```

Reply:
If we're fixing typos :)
Your changes all look fine though

Reply:
Dang, good catch!